projects/packages/backup/package.json

```diff
@@ -65,4 +65,4 @@
 		"webpack": "5.94.0",
 		"webpack-cli": "6.0.1"
 	}
-}
+}
```

Comment:
Why remove the newline from the end of the file?

Same in other files below.

Reply:
I am not sure. It's not flagged as a violation for `package.json` files, apparently. I have my editor set up to apply linting when saving, so these get removed (I think the root issue is elsewhere though). Something to do with the lining configuration perhaps, cause it's not the case in Calypso (for example - `.json` files are parsed with `eol-last` like regular JS).

Reply:
We don't have any eslint set up for package.json files. Yet, anyway.

Probably your editor or something saves them without the trailing newline, then in Calypso linting fixes it.